### PR TITLE
Remove an `examples` directory

### PR DESCRIPTION
**Pull request by c-bata**
## Motivation

The readme file in`examples` directory is not maintained and has differences from the readme file in https://github.com/optuna/optuna-examples.

## Description of the changes

Remove `examples/README.md`.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -1,120 +1,5 @@
 Optuna Examples
 ================
 
-This page contains a list of example codes written with Optuna. The example files are in [optuna/optuna-examples](https://github.com/optuna/optuna-examples/).
+The example files were moved to [optuna/optuna-examples](https://github.com/optuna/optuna-examples/).
```
Comment (hvy) [inline examples/README.md:4]: Nit, really.
```suggestion
Example files have been moved to [optuna/optuna-examples](https://github.com/optuna/optuna-examples/).
```

Comment (hvy) [inline examples/README.md:4]: Nit, really.
```suggestion
Example files have been moved to [optuna/optuna-examples](https://github.com/optuna/optuna-examples/).
```

**Review by hvy — APPROVED** [review]
LGTM!

**Comment by c-bata** [comment]
I applied your suggestion 👍  PTAL.

**Review by HideakiImamura — APPROVED** [review]
LGTM.

**Pull request merged**